skydriver/__main__.py

```diff
@@ -2,13 +2,31 @@
 
 import asyncio
 
-from . import server
-from .config import ENV, config_logging
+from . import database, k8s, server
+from .config import ENV, LOGGER, config_logging
 
 
 async def main() -> None:
-    """Start server."""
-    rs = await server.make()
+    """Establish connections and start components."""
+
+    # Mongo client
+    LOGGER.info("Setting up Mongo client...")
+    mongo_client = await database.create_mongodb_client()
+    LOGGER.info("Mongo client connected.")
+
+    # K8s client
+    LOGGER.info("Setting up k8s client...")
+    k8s_api = k8s.setup_k8s_client()
+    LOGGER.info("K8s client connected.")
+
+    # Scan Backlog Runner
+    LOGGER.info("Starting scan backlog runner...")
+    asyncio.create_task(k8s.scan_backlog.startup(k8s_api, mongo_client))
```

Comment:
It would be good to keep track of this task, so when the server stops we cancel it cleanly.

Reply:
when does the server stop? I can only think of when the container is terminated by argocd, which I've assumed is a system exit, so there's no in-process clean up

Reply:
I guess an exception could stop it?  Otherwise you're probably right.

Reply:
That's true. This does broadly bring up the problem of how to resume after a restart